titan/features/prep.py

```diff
@@ -134,7 +134,7 @@ def get_acquisition_risk_multiplier(self, time: int, interaction_type: str):
                     return 1.0 - params.prep.efficacy.adherent
                 else:
                     return 1.0 - params.prep.efficacy.non_adherent
-            elif self.type == "Inj" and self.adherent == False:
+            elif self.type == "Inj" and self.adherent is False:
```

Comment:
I know I already commented on this to be is False... but it should really be `not self.adherent`... sorry!

Reply:
Ahh, I should've caught that!